integration/kubernetes/k8s-volume.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+
+setup() {
+	export KUBECONFIG=/etc/kubernetes/admin.conf
+	pod_config_dir="${BATS_TEST_DIRNAME}/untrusted_workloads"
+	path="/mnt/data"
```

Comment:
Might be better (safer from a whole system point of view) if we made this path a mktmpdir rather than hard wire it to a subdir of /mnt